### DISCUSSION TITLE
How does homebrew handle cask apps that update themselves?

### DISCUSSION THREAD

**Discussion by Danathar** [Homebrew/discussions#3970] [Casks]
Before I take the plunge on installing things like Steam, Zoom and Firefox via homebrew via app casks, can anybody shed some light on what happens if one of these apps self-update. Steam does that a lot on it's own, and Zoom is now set to auto update. What happens when they do in regards to the version that homebrew thinks it has? If you do a `brew update` will it overwrite the version the app auto updated itself to? Whats the best practice in this regard?

thanks!

**Comment by SMillerDev** [comment]
https://github.com/orgs/Homebrew/discussions/3359 has an explanation for this already.

**Reply by Danathar** [reply]
Thanks! I guess when I searched I didn’t craft my search right.

**Reply by ethanbeyer** [reply]
I'm glad you opened this discussion because I wasn't able to craft my search right, either!
